src/certificate/CertificatePayloadGenerator.ts

```diff
@@ -47,7 +49,8 @@ export class CertificatePayloadGenerator implements ICertificatePayloadCommand {
 		@Inject() private testHistoryCommand: TestHistoryCommand,
 		@Inject() private defectsCommand: DefectsCommand,
 		@Inject() private makeAndModelCommand: MakeAndModelCommand,
-		@Inject() private odometerHistoryCommand: OdometerHistoryCommand
+		@Inject() private odometerHistoryCommand: OdometerHistoryCommand,
```

Comment:
Should this be named abandonedCertGenerator instead of command? To match the rest of the cert generators?

Reply:
Done and pushed.